lib/diagnostic.js

```diff
@@ -46,7 +46,7 @@ let person2 = {
 };
 
 // Question 3:  Invoke the `fullName()` method so it returns `Santa Clause`?
-let answer3 = null; // replace null with your response
+let answer3 = this.fullName; // replace null with your response
```

Comment:
`this.fullName;` would return undefined. 

You need to call `person1`'s `fullName()` method but use `Function`'s `call()` method to call it in the context of `person2`.

```javascript
person1.fullName.call(person2);
```

Reach out for a 1on1 if you want some more clarification.